tools/ci/ci-packages.txt

```diff
@@ -25,82 +21,76 @@ icewm-default-1.4.2
 icewm-theme-branding-1.2.5
 jq-1.6
 libaspell15-0.60.8
-libatopology2-1.2.4
-libbrotlicommon1-1.0.7
-libbrotlidec1-1.0.7
+libatopology2-1.2.6.1
 libcairo-gobject2-1.16.0
-libcmark0_28_3-0.28.3
-libcolord2-1.4.4
-libcroco-0_6-3-0.6.13
+libcmark0_30_2-0.30.2
+libcolord2-1.4.5
 libdialog14-1.3
-libepoxy0-1.4.3
+libepoxy0-1.5.9
 libfontenc1-1.1.3
-libgdk_pixbuf-2_0-0-2.40.0
-libgtk-3-0-3.24.20
+libgdk_pixbuf-2_0-0-2.42.6
+libgdk_pixbuf_xlib-2_0-0-2.40.2
+libgtk-3-0-3.24.31
 libimagequant0-2.11.3
 libjq1-1.6
-libjson-glib-1_0-0-1.4.4
-liblcms2-2-2.9
+liblcms2-2-2.12
 libonig4-6.7.0
 libpci3-3.5.6
-librest-0_7-0-0.8.1
-librsvg-2-2-2.46.5
+librsvg-2-2-2.52.6
 libsamplerate0-0.1.9
-libsoup-2_4-1-2.68.3
 libssh2-1-1.9.0
-libwayland-cursor0-1.18.0
 libXaw7-1.0.13
 libXcursor1-1.1.15
 libXdmcp6-1.1.2
 libXfont2-2-2.0.3
-libXi6-1.7.9
 libXinerama1-1.1.3
 libxkbfile1-1.0.9
 libXmu6-1.1.2
 libXmuu1-1.1.2
+libXpm4-3.5.12
 libXss1-1.2.2
 libXt6-1.1.5
-libXvnc1-1.9.0
+libXvnc1-1.10.1
 lsof-4.91
 luit-20150706
 optipng-0.7.7
 pciutils-3.5.6
-perl-Algorithm-C3-0.10
+perl-Algorithm-C3-0.11
 perl-Algorithm-Diff-1.1903
 perl-aliased-0.34
 perl-AppConfig-1.71
-perl-App-cpanminus-1.7045
+perl-App-cpanminus-1.7044
```

Comment:
Hm, that's a downgrade.
https://metacpan.org/dist/App-cpanminus/changes#L7
Did we have the module [here](https://build.opensuse.org/project/subprojects/devel:openQA) before?

Reply:
this can also happen if Leap 15.3 has a maintenance update and Leap 15.4 does not have it yet.